app/routes/note/note.controller.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by gaston on 8/9/16.
+ */
+function NoteController(urlParams){
+    let self = this;
+    NavBar.setTitle("Nota");
```

Comment:
Where Navbar was set? We should use global vars. What about imported it? We could create a singleton class which return nabvar instance. We can talk later.